azuredevops/internal/acceptancetests/resource_workitemquery_test.go

```diff
@@ -0,0 +1,128 @@
+package acceptancetests
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/acceptancetests/testutils"
+)
+
+// Query directly under area (Shared Queries)
+func TestAccWorkItemQuery_UnderArea(t *testing.T) {
+	projectName := testutils.GenerateResourceName()
+	queryName := "tfacc-query-area"
+	wiql := "SELECT [System.Id] FROM WorkItems WHERE [System.TeamProject] = @project"
+
+	config := hclWorkItemQueryResource(projectName, queryName, wiql)
+
+	res := "azuredevops_workitemquery.query"
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testutils.PreCheck(t, nil) },
+		Providers:    testutils.GetProviders(),
+		CheckDestroy: testutils.CheckProjectDestroyed,
+		Steps: []resource.TestStep{{
+			Config: config,
+			Check: resource.ComposeTestCheckFunc(
+				testutils.CheckProjectExists(projectName),
+				resource.TestCheckResourceAttr(res, "name", queryName),
+				resource.TestCheckResourceAttr(res, "wiql", wiql),
+			),
+		}},
```

Comment:
Can we add an import step after each "regular" test step to validate the import functionality also works?

(This applies to every test case)

Refer to the *azuredevops/internal/acceptancetests/resource_project_test.go* for an example.